beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -726,18 +720,19 @@ func (s *Server) GetAttesterSlashingsV2(w http.ResponseWriter, r *http.Request)
 	ctx, span := trace.StartSpan(r.Context(), "beacon.GetAttesterSlashingsV2")
 	defer span.End()
 
+	slot := s.HeadFetcher.HeadSlot()
```

Comment:
Head slot is the slot of the most recent block, not the current slot. There is a `TimeFetcher` interface that can give you the current slot.